salt/states/boto3_route53.py

```diff
@@ -83,6 +84,11 @@ def __virtual__():
     '''
     return 'boto3_route53' if 'boto3_route53.find_hosted_zone' in __salt__ else False
 
+def _ensure_unicode(obj):
```

Comment:
I'm pretty sure the `salt.utils.stringutils.to_unicode` function properly handles this.